unic/ucd/block/tests/basic_tests.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The UNIC Project Developers.
+//
+// See the COPYRIGHT file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+extern crate unic_ucd_block;
+use unic_ucd_block::{Block, BlockIter};
+
+#[test]
+fn test_block_of() {
+    let latin_1 = Block::of('\u{00a9}').unwrap(); // ©, COPYRIGHT SIGN
+    assert_eq!(latin_1.range.low, '\u{0080}');
+    assert_eq!(latin_1.range.high, '\u{00ff}');
+    assert_eq!(latin_1.name, "Latin-1 Supplement");
+}
+
+#[test]
+fn test_block_iter() {
+    assert_eq!(BlockIter::new().count(), 277);
```

Comment:
Just found out that `char` can't represent surrogate code points. So there are 3 blocks missing when we generate the tables (Unicode 10.0 has 280 assigned blocks).

```
D800..DB7F; High Surrogates
DB80..DBFF; High Private Use Surrogates
DC00..DFFF; Low Surrogates
```

Reply:
Right. And that's fine, since non of these codepoints are valid characters anymore. (They were only valid before introduction of UTF-16.)